keyboards/centromere/centromere.h

```diff
@@ -38,10 +40,26 @@
   k2a, k20, k21, k22, k23, k24,      k25, k26, k27, k28, k29, k2b,\
                  k32, k33, k34,      k35, k36, k37  \
 ) \
-  {                                                           \
-    { k00,     k01, k02, k03, k04, k05, k06, k07, k08,   k09   }, \
-    { k10,     k11, k12, k13, k14, k15, k16, k17, k18,   k19   }, \
-    { k20,     k21, k22, k23, k24, k25, k26, k27, k28,   k29   }, \
-    { KC_NO, KC_NO, k32, k33, k34, k35, k36, k37, KC_NO, KC_NO }, \
-    { KC_NO, KC_NO, k2a, k1a, k0a, k0b, k1b, k2b, KC_NO, KC_NO }  \
+  {                                                       \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09 }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19 }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29 }, \
+    { ___, ___, k32, k33, k34, k35, k36, k37, ___, ___ }, \
+    { ___, ___, k2a, k1a, k0a, k0b, k1b, k2b, ___, ___ }  \
   }
+
+#define LAYOUT_mini( \
+  k00, k01, k02, k03, k04,      k05, k06, k07, k08, k09, \
+  k10, k11, k12, k13, k14,      k15, k16, k17, k18, k19,\
+  k20, k21, k22, k23, k24,      k25, k26, k27, k28, k29,\
+            k32, k33, k34,      k35, k36, k37  \
+) \
+  {                                                       \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09 }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19 }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29 }, \
+    { ___, ___, k32, k33, k34, k35, k36, k37, ___, ___ }, \
+    { ___, ___, ___, ___, ___, ___, ___, ___, ___, ___ }  \
+  }
+
+#define LAYOUT_split_3x5_3 LAYOUT_mini
```

Comment:
I would prefer to see these swapped around. Same for the other PR.

Reply:
Sorry, but I don't understand. What do you want to see swapped around?

Reply:
I think @fauxpark means that they want `LAYOUT_split_3x5_3` for the main macro name above.